tests/E2E/test/campaigns/full/10_module/2_cross_selling_module.js

```diff
@@ -1 +1,62 @@
-require('../../high/10_module/2_cross_selling_module');
+const {AccessPageBO} = require('../../../selectors/BO/access_page');
```

Comment:
Duplicating code will prevent you to maintain your code efficiently later.

Reply:
Ok, deleted from this PR and will do it in other one